DependencyInjection/DoctrineExtension.php

```diff
@@ -250,6 +250,12 @@ protected function loadDbalConnection($name, array $connection, ContainerBuilder
 
         unset($connection['auto_commit']);
 
+        if (isset($connection['disable_type_comments'])) {
+            $configuration->addMethodCall('setDisableTypeComments', [$connection['disable_type_comments']]);
+        }
+
+        unset($connection['disable_type_comments']);
```

Comment:
could also go inside the `if` statement above I guess?

Reply:
Ah nevermind. It can be null.

Reply:
I kept it consistent with other parameters than need to be removed from the config passed to the connection factory.